src/toplevel/learnocaml_toplevel_pp.ml

```diff
@@ -0,0 +1,28 @@
+(* This file is part of Learn-OCaml.
+ *
+ * Copyright (C) 2019 OCaml Software Foundation.
+ * Copyright (C) 2016-2018 OCamlPro.
+ *
+ * Learn-OCaml is distributed under the terms of the MIT license. See the
+ * included LICENSE file for details. *)
+
+(* Code to build an image *)
```

Comment:
You should be a bit more specific here: you are not building an image (as it is taken as input); instead, you are rendering an image as a bitmap.

It should also be useful to understand why the size of the output bitmap is not taken as arguments of this function.